content/modules/upgrading-rs.md

```diff
@@ -70,27 +70,31 @@ When you upgrade the module for a database, you can either:
 1. To upgrade the module for the database, run:
 
     ```src
-    rladmin upgrade module db_name <database_name> module_name <module_name> version <new_module_version_number> module_args <module arguments>
+    rladmin upgrade db <database_name> and module module_name <module_name> version <new_module_version_number> module_args "<module arguments>"
     ```
 
+    - If you want to remove the existing module arguments, enter `""` without arguments.
+    - If you want to use the exisintg module arguments, enter `"keep_args"`
```

Comment:
This could just be me but I find the whole quotation mark thing so weird (it's really not the command line standard) that I would stress "with the quotation marks" or something. But again, this is just my personal take. I'm not a writer.

Reply:
It's in code style and we have examples so I think it will be OK.